roles/devscripts/tasks/133_host_network.yml

```diff
@@ -56,6 +56,11 @@
         list
       }}
   block:
+    - name: Ensure firewalld is installed
```

Comment:
that one was needed - in molecule I was changing the value of the `manage_br_bridge` and hit a failure in the molecule job because nothing installs firewalld.

Better install it twice than never - this is why it's also in this patch.